test/src/test/java/org/corfudb/integration/LogReplicationDynamicStreamIT.java

```diff
@@ -0,0 +1,520 @@
+        verifyDataOnStandby(mapTagOneStandby, numWrites);
+
+        streamingStandbySnapshotCompletion.await();
+        numTxLatch.await();
+        assertThat(listener.messages.size()).isEqualTo(expectedMessageSize);
+    }
+
+    /**
+     * This test will verify the stream listener could correctly receive the updates during log entry sync.
+     *
+     * (1) Perform a basic snapshot sync and log entry sync, verify the cluster is in log entry sync state
+     * (2) Open a new map on both Source and Sink with TAG_ONE
+     * (3) Initiate a testing stream listener and subscribe to TAG_ONE on Sink
+     * (4) Write some new entries to this new map on Source
+     * (5) Verify new data get replicated and the stream listener received all the updates
+     */
+    @Test
+    public void testStandbyStreamingLogEntrySync() throws Exception {
+        // perform basic snapshot and log entry sync, the cluster should be in log entry sync state now
+        testSnapshotAndLogEntrySync();
+
+        // Verify log replication status is log entry sync
+        verifyInLogEntrySyncState();
+
+        // Open testing map on Source and Sink
+        String streamName = "TableStreaming001";
+
+        Table<StringKey, IntValue, Metadata> mapTagOne = corfuStoreActive.openTable(
+                NAMESPACE,
+                streamName,
+                StringKey.class,
+                IntValue.class,
+                Metadata.class,
+                TableOptions.fromProtoSchema(ValueFieldTagOne.class)
+        );
+
+        Table<StringKey, IntValue, Metadata> mapTagOneStandby = corfuStoreStandby.openTable(
+                NAMESPACE,
+                streamName,
+                StringKey.class,
+                IntValue.class,
+                Metadata.class,
+                TableOptions.fromProtoSchema(ValueFieldTagOne.class)
+        );
+
+        // Subscribe the testing stream listener
+        UUID streamId = CorfuRuntime.getStreamID(mapTagOne.getFullyQualifiedTableName());
+        CountDownLatch streamingStandbySnapshotCompletion = new CountDownLatch(numWrites);
+        CountDownLatch numTxLatch = new CountDownLatch(numWrites);
+        StreamingStandbyListener listener = new StreamingStandbyListener(streamingStandbySnapshotCompletion,
+                numTxLatch, Collections.singleton(streamId));
+        corfuStoreStandby.subscribeListener(listener, NAMESPACE, DefaultLogReplicationConfigAdapter.TAG_ONE);
+
+        writeToMap(mapTagOne, true, 0, numWrites);
+
+        // Verify snapshot sync succeeded and stream listener received all the changes
+        verifyDataOnStandby(mapTagOneStandby, numWrites);
+
+        streamingStandbySnapshotCompletion.await();
+        numTxLatch.await();
+        assertThat(listener.messages.size()).isEqualTo(numWrites);
+    }
+
+    /**
+     * This test will verify the streams to stream tags map is correctly rebuilt during snapshot sync.
```

Comment:
I think this comment needs to be updated?  It seems that we are testing if the new table was added to the registry table on the Sink?

Reply:
elaborated it a little bit